SI/Sem.07/Pract.07/Strings/Task08.cpp

```diff
@@ -0,0 +1,50 @@
+//solution without counting each symbol
+
+#include <iostream>
+const int MAX_SIZE = 100;
+const int SIZE_SYMBOLS = 128;
+
+//void CountSymbols(char* symbols, int* symbolsCount)
+//{
+//	for (int i = 0; i < MAX_SIZE; i++)
+//	{
+//		if (symbols[i] == '\0')
+//		{
+//			break;
+//		}
+//		symbolsCount[symbols[i]]++;
+//	}
+//}
+void PrintWithoutRepeating(char* str)
+{
+	int symbolsCount[SIZE_SYMBOLS] = {};
+	//CountSymbols(str, symbolsCount);
+
+	char newStr[MAX_SIZE] = {};
+	int indexForNewStr = 0;
+
+	for (int i = 0; i < MAX_SIZE; i++)
+	{
+		if (str[i] == '\0')
+		{
```

Comment:
It is a matter of style, but this looks odd - `for (int i = 0; i < MAX_SIZE && str[I] != '\0'; i++)` or maybe a while loop with a counter would look better